plenum/test/consensus/order_service/sim_helper.py

```diff
@@ -25,14 +25,22 @@ def setup_consensus_data(cd):
     cd.node_mode = Mode.participating
 
 
-def setup_pool(random):
+def update_config(config, updated_args: dict):
```

Comment:
As config is a global singleton, we need to make sure that changes values are reverted after the test is finished. 
Have a look at context managers and helper functions we have (such as `view_change_timeout` or `tconf` fixtures with yield).